doc/development/setup.rst

```diff
@@ -54,6 +55,12 @@ Working with the code
 The first thing you need are all the main application's dependencies::
```

Comment:
I'd explicitly call out nodejs in case people want to install it a different way or already have it installed

```suggestion
The first thing you need are a recent ``nodejs`` installation and all the main application's Python-level dependencies::
```